events/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class Event(models.Model):
+
+    class Colors(models.TextChoices):
+        RED = "#FF0000", "Red"
+        PINK = "#FFC0CB", "Pink"
+        ORANGE = "#FFA500", "Orange"
+        YELLOW = "#FFFF00", "Yellow"
+        PURPLE = "#800080", "Purple"
+        GREEN = "#008000", "Green"
+        BLUE = "#0000FF", "Blue"
+        BROWN = "#A52A2A", "Brown"
+        WHITE = "#FFFFFF", "White"
+        GRAY = "#808080", "Gray"
+        BLACK = "#000000", "Black"
+
+    title = models.CharField(max_length=50)
+    location = models.CharField(blank=True, max_length=50)
+    description = models.TextField(blank=True)
+    date_time_start = models.DateTimeField()
+    date_time_end = models.DateTimeField()
+    color = models.CharField(
+        max_length=7,
+        choices=Colors.choices,
+        default=Colors.BLACK,
+    )
+
+    def clean(self):
+        if not self.title:
+            raise ValidationError('title cannot be blank')
+        if not self.date_time_start:
+            raise ValidationError('Starting date cannot be blank')
+        if not self.date_time_end:
+            raise ValidationError('Ending date cannot be blank')
+        if self.date_time_start >= self.date_time_end:
+            raise ValidationError(f'{self.date_time_start} must be smaller than {self.date_time_end}')
+
+    def save(self, *args, **kwargs):
+        self.clean()
```

Comment:
AFAIU this is a validation method so maybe you can call it something like:
```suggestion
        self.validate_params()
```

Reply:
![Screenshot_3](https://user-images.githubusercontent.com/83750345/160833971-7e6575ec-6ec2-499f-894a-b3e197b187cf.png)

changing the methods name breaks the admin page, from what i understood from django documentation
django expects validation to be inside a clean method

Reply:
@BelikovYoav please take a look at the error in the "Exception Value"
btw I download your PR, changed it locally, run vagrant up... and add an event without any problem

Reply:
@kobihk the problem occurs when you try to add an event with date time start bigger than date time end, not when you try to add a "good" event

Reply:
@BelikovYoav I see 
Thanks!!